src/content/reference/react-dom/server/renderToStaticMarkup.md

```diff
@@ -16,62 +16,62 @@ const html = renderToStaticMarkup(reactNode)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `renderToStaticMarkup(reactNode)` {/*rendertostaticmarkup*/}
 
-On the server, call `renderToStaticMarkup` to render your app to HTML.
+Pada *server*, panggil fungsi `renderToStaticMarkup` untuk me-*render* aplikasi Anda sebagai HTML.
```

Comment:
```suggestion
Pada *server*, panggil fungsi `renderToStaticMarkup` untuk me-*render* aplikasi Anda ke HTML.
```

Just need to fix this because the source is `to`, not `as`.